.github/workflows/instant_benchmark.yml

```diff
@@ -31,6 +31,10 @@ on:
         description: 'The container used to run benchmark (overrides the template). Should be a full docker path such as deepjavalibrary/djl-serving:0.28.0-lmi'
         required: false
         default: ''
+      ecr-login:
```

Comment:
This is only in workflow_dispatch, but would also have to be in workflow_call to be used from other workflows